scripts/zones/Abyssea-Grauberg/npcs/qm_raja_3.lua

```diff
@@ -1,6 +1,6 @@
 -----------------------------------
 -- Zone: Abyssea-Grauberg
---  NPC: qm_raja_3 (???)
+--  NPC: qm_raja_3(???)
```

Comment:
Please remove space before **NPC:** (so that it aligns with the start of the other lines) and please add a space between the **qm_raja_3** and the opening parenthesis